Tests/LeafTests/BodyWhitespaceTests.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import XCTest
+@testable import Leaf
+
+class BodyWhitespaceTests: XCTestCase {
```

Comment:
We'll need to add these to `Tests/LinuxMain.swift` so they run properly, look at the pattern in that file for other tests and you should be a-ok.